### PR TITLE
azure-cli 2.0.19

### PR DESCRIPTION
**Pull request by derekbekoe**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Currently blocked due to https://github.com/Azure/azure-batch-cli-extensions/issues/37
Closes https://github.com/Azure/azure-cli/issues/4642

### CODE REVIEW

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

File: Formula/azure-cli.rb [hunk 1]
```diff
@@ -36,8 +36,13 @@ class AzureCli < Formula
...
 
   resource "azure-batch" do
-    url "https://files.pythonhosted.org/packages/91/00/710e92aa69e3938c46c4bd3e61a24df2904cfda70fea99f6c68485e99f4a/azure-batch-3.1.0.zip"
-    sha256 "a4446696392105222abe14df07d6b174572f245aa55f86fa5fc55eb3a7186900"
+    url "https://files.pythonhosted.org/packages/4e/24/b6764cbb01ea09a1a8f464869ecc926cbc1d4e6dca580b2d16c974e41449/azure-batch-4.0.0.zip"
+    sha256 "35c617cb601dd286ea6d078f95e0c6b159b4c7974ed6f3693c6dffd2513285af"
+  end
+
+  resource "azure-batch-extensions" do
+    url "NOTFOUND"
```
Comment (fxcoudert) [inline Formula/azure-cli.rb]: What are you expecting to happen here?

Comment (derekbekoe) [reply Formula/azure-cli.rb]: `azure-batch-extensions` to publish the source on PyPI which has been done now.

**Comment by ilovezfs** [comment]
Thanks for the upgrade @derekbekoe! :white_check_mark:

**Pull request closed without merging**